harvard-university-for-the-creative-arts.csl

```diff
@@ -398,6 +458,13 @@
         </group>
         <text macro="legal-detail"/>
         <text macro="locator"/>
+        <group>
+          <choose>
+            <if type="map" match="any">
+              <text variable="collection-title" prefix="(" suffix=")"/>
+            </if>
+          </choose>
+        </group>
```

Comment:
You should be able to replace

```xml
        <group>
          <choose>
            <if type="map" match="any">
              <text variable="collection-title" prefix="(" suffix=")"/>
            </if>
          </choose>
        </group>
```

with

```xml
        <choose>
          <if type="map" match="any">
            <text variable="collection-title" prefix="(" suffix=")"/>
          </if>
        </choose>
```

(you need to delete both the starting `<group>` and closing `</group>` elements)